fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/impl/AsyncDruidWebServiceImplV2.java

```diff
@@ -0,0 +1,71 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.druid.client.impl;
+
+import com.yahoo.bard.webservice.druid.client.DruidServiceConfig;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.MappingJsonFactory;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import org.asynchttpclient.Response;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.function.Supplier;
+
+import javax.ws.rs.core.Response.Status;
+
+/**
+ * Represents the druid web service endpoint for partial data V2.
+ * <p>
+ * When Druid returns data with missing intervals with a non-OK status code, instead of
```

Comment:
Will Druid have a non-OK response code in that case? (ie. where there are missing intervals) Or does it still return as a 200 OK?